crates/xetex_layout/src/lib.rs

```diff
@@ -27,6 +27,175 @@ mod linkage {
     use tectonic_bridge_icu as clipyrenamehack5;
 }
 
+macro_rules! cstr {
```

Comment:
Hi. [CStr literals](https://rust-lang.github.io/rfcs/3348-c-str-literal.html) have been stabilized recently so they'd fit in nicely here :)

Reply:
It looks like they were delayed by a release, so they're stabilizing in 1.77, the release after the one coming out in a couple days. I'll leave a TODO to fix it then, if this merges before that release.

Reply:
Oh whoops, right. "Tracking issue closed as stabilized" doesn't mean that the version is stable already ^^

Reply:
Also, we don't have a formal Minimum Supported Rust Version (MSRV) policy, but I think it's generally good to give new stabilizations at least a few features to propagate through the ecosystem before starting to rely on them.